### PR TITLE
Add a banner to login page

### PR DESCRIPTION
**Pull request by hmhsu**
### Description
Add a banner to login page.
Styles: https://play.tailwindcss.com/6yyHmMFEmG?size=540x720
Screen Shot 2021-03-23 at 12 38 12 PM

### Checklist
- [ ] Created tests which fail without the change (if possible)

### Issue(s)
#555

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 1]
```diff
@@ -55,6 +61,57 @@ public Content render(Http.Request request, Optional<String> message) {
...
+                                    Styles.W_0, Styles.FLEX_1, Styles.FLEX, Styles.ITEMS_CENTER)
+                                .with(
+                                    span()
+                                        .withClasses(
+                                            Styles.FLEX,
+                                            Styles.P_2,
+                                            Styles.ROUNDED_LG,
+                                            Styles.BG_RED_800)
+                                        .with(
+                                            new ContainerTag("svg")
```
Comment (AWBaumann) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:93]: This should work:

```
Icons.svg(Icons.LOGIN_BANNER_PATH, 24)
    .withClasses(Styles.H_6, Styles.W_6, Styles.TEXT_WHITE)
    .attr("fill", "none")
    .attr("stroke-linecap", "round")
    .attr("stroke-linejoin", "round")
    .attr("stroke-width", "2")
```

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 2]
```diff
@@ -55,6 +61,57 @@ public Content render(Http.Request request, Optional<String> message) {
...
+                                                        .attr("stroke-linejoin", "round")
+                                                        .attr("stroke-width", "2")
+                                                        .attr("d", Icons.LOGIN_BANNER_PATH))),
+                                    p().withClasses(
+                                            Styles.ML_3,
+                                            Styles.FONT_MEDIUM,
+                                            Styles.TEXT_WHITE,
+                                            Styles.TRUNCATE)
+                                        .with(
+                                            span(BANNER_TEXT).withClasses("md:hidden"),
```
Comment (AWBaumann) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:113]: I don't think you need both spans. 

try `withClasses(StylesUtils.responsiveMedium(Styles.INLINE)`

...but that still shouldn't be necessary as span should default to inline.

Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:113]: Thanks for pointing out. I was just blindly copying the styles over.

File: universal-application-tool-0.0.1/app/views/LoginForm.java (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/LoginForm.java]: Done.

File: universal-application-tool-0.0.1/app/views/LoginForm.java (hunk 2 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/LoginForm.java]: Done.

**Review by AWBaumann — APPROVED** [review]

**Pull request merged**